src/EditorFeatures/CSharpTest/CodeActions/EncapsulateField/EncapsulateFieldTests.cs

```diff
@@ -1387,7 +1387,7 @@ public int I
 }", options: Option(CodeStyleOptions.QualifyFieldAccess, true, NotificationOption.Error));
         }
 
-        [Fact, Trait(Traits.Feature, Traits.Features.EncapsulateField), Test.Utilities.CompilerTrait(Test.Utilities.CompilerFeature.Tuples)]
```

Comment:
These changes were needed as part of this PR due to C# editorfeatures test project now referencing core editorfeatures test project (was causing compile errors otherwise).